images/linux/scripts/base/apt-sources-list-arch.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+################################################################################
+##  File:  apt-sources-list-arch.sh
+##  Desc:  Limit apt sources to the actual architectures
+################################################################################
+
+sudo sed -i'' -E 's/^deb http:\/\/(azure.archive|security).ubuntu.com/deb [arch=amd64,i386] http:\/\/\1.ubuntu.com/' /etc/apt/sources.list
```

Comment:
Do we really need the separate script for one line?
I suggest to put it to [apt.sh](https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/base/apt.sh) or [repos.sh](https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/base/repos.sh)